### PR TITLE
Ruby linting gems

### PR DESCRIPTION
**Pull request by xjunior**
- Create rubocop-powerhome extension
- Introduce rubocop-cobra plugin with all cobra specific cops

### CODE REVIEW

File: .github/workflows/rubocop-cobra.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.ruby }}
+    strategy:
+      matrix:
+        ruby:
+          - '2.7.4'
```
Comment (benlangfeld) [inline .github/workflows/rubocop-cobra.yml:13]: Can we test on Ruby 3.1 also? Can be in a follow-up PR.

File: rubocop-powerhome/rubocop-powerhome.gemspec [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative "lib/rubocop/powerhome/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "rubocop-powerhome"
+  spec.version = RuboCop::Powerhome::VERSION
+  spec.authors = ["Carlos Palhares"]
+  spec.email = ["chjunior@gmail.com"]
```
Comment (benlangfeld) [inline rubocop-powerhome/rubocop-powerhome.gemspec:9]: Please use your Power email address.

Comment (xjunior) [inline rubocop-powerhome/rubocop-powerhome.gemspec:9]:
```suggestion
  spec.email = ["carlos.palhares@powerhrg.com"]
```

File: rubocop-cobra/rubocop-cobra.gemspec [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative "lib/rubocop/cobra/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "rubocop-cobra"
+  spec.version = RuboCop::Cobra::VERSION
+  spec.authors = ["Carlos Palhares"]
+  spec.email = ["chjunior@gmail.com"]
```
Comment (xjunior) [inline rubocop-cobra/rubocop-cobra.gemspec:9]:
```suggestion
  spec.email = ["carlos.palhares@powerhrg.com"]
```

**Review by benlangfeld — DISMISSED** [review]

**Review by wadewinningham — DISMISSED** [review]

**Review by wadewinningham — APPROVED** [review]

**Pull request merged**